frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/tags/DisplayStateBoxes.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright Siemens AG, 2017.
+ * Part of the SW360 Portal Project.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.sw360.portal.tags;
+
+import org.eclipse.sw360.datahandler.common.ThriftEnumUtils;
+import org.eclipse.sw360.datahandler.thrift.projects.Project;
+import org.eclipse.sw360.datahandler.thrift.projects.ProjectState;
+
+
+import javax.servlet.jsp.JspException;
+import javax.servlet.jsp.tagext.JspTag;
+import javax.servlet.jsp.tagext.SimpleTagSupport;
+import java.io.IOException;
+
+import static org.eclipse.sw360.datahandler.thrift.projects.ProjectClearingState.*;
+
+public class DisplayStateBoxes extends SimpleTagSupport {
+
+    public static final String PROJECT_STATE_ACTIVE__CSS      = "projectStateActive";
```

Comment:
String also used in `frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/tags/DisplayProjectStateBox.java`. Please move constant to the next general place and use it in both places.